src/main/java/org/mozilla/msrp/platform/user/UserController.java

```diff
@@ -103,11 +104,14 @@ void login(@RequestParam(value = "code") String code,
                 log.info("[login]bind success: login more than three times. User suspended");
                 httpResponse.sendRedirect("/api/v1/done?jwt=" + customToken + "&login_success=true&disabled=true&times=3");
             } else if (loginResponse instanceof LoginResponse.Fail) {
-                log.info("[login]bind fail:" + ((LoginResponse.Fail) loginResponse).getMessage());
+                final String message = ((LoginResponse.Fail) loginResponse).getMessage();
+                Metrics.event(Metrics.EVENT_USER_BIND_FAIL, message);
+                log.info("[login]bind fail:" + message);
                 httpResponse.sendRedirect("/api/v1/done?login_success=false");
             }
 
         } catch (IOException | JSONException e) {
+            Metrics.event(Metrics.EVENT_USER_BIND_FAIL, "Error");
```

Comment:
not sure where the JSONException may come from, but I think json exception is worth being logged?

Reply:
fixed